src/datasets/info.py

```diff
@@ -93,23 +93,24 @@ class DatasetInfo:
     Note: Not all fields are known on construction and may be updated later.
 
     Attributes:
-        description (str):
-        citation (str):
-        homepage (str):
-        license (str):
-        features (Features, optional):
+        description (str): A description of the dataset.
+        citation (str): A BibTeX citation of the dataset.
+        homepage (str): A URL to the official homepage for the dataset.
+        license (str): The dataset's license.
+        features (Features, optional): The features used to specify the dataset's columns, types and conversion methods.
         post_processed (PostProcessedInfo, optional):
```

Comment:
```suggestion
        post_processed (PostProcessedInfo, optional): Info regarding the resources of a possible post-processing of a dataset. It can contain the information of an index for example.
```